modules/geolocation/geolocation.py

```diff
@@ -285,7 +315,13 @@ def run(
             )
             # Partial data not allowed
             if not result:
+                frame = inspect.currentframe()
+                self.__logger("could not convert detection to world from image", frame)
                 return False, None
             detections_in_world.append(detection_world)
 
+        for detection_in_world in detections_in_world:
+            frame = inspect.currentframe()
+            self.__logger.info(detection_in_world, frame)
```

Comment:
we should log each one as it's created in the for loop above (after we append it to the list